CMakeLists.txt

```diff
@@ -76,9 +76,13 @@ install(
 
 # make find_package(gridformat) work when used from build directory
 if (GRIDFORMAT_HIGHFIVE_SOURCE_INCLUDED)
+    configure_file(
+        ${CMAKE_CURRENT_SOURCE_DIR}/deps/HighFive/cmake/HighFiveConfig.cmake
+        ${CMAKE_CURRENT_BINARY_DIR}/cmake/HighFiveConfig.cmake
+    )
     export(
         EXPORT HighFiveTargets
-        FILE "${CMAKE_CURRENT_BINARY_DIR}/cmake/HighFiveTargets.cmake"
+        FILE "${CMAKE_CURRENT_SOURCE_DIR}/HighFiveTargets.cmake"
```

Comment:
I had to install the targets into the root source dir, because in the highfive config file it looks for the targets in the currently processed list dir: https://github.com/highfive-devs/highfive/blob/2c2644a75956894cd4d436be9ec2e52a23b76d7a/cmake/HighFiveConfig.cmake#L12